utils/index.js

```diff
@@ -18,3 +18,22 @@ export const getAutoSuggestUsers = async (loginSubstring, limit) => {
         return { error: error.message };
     }
 };
+
+export const addUsersToGroup = async (groupId, userId) => {
```

Comment:
Requirement says `Add addUsersToGroup(groupId, userIds)method which will allow adding users to a certain group`

The second parameter should be arrays of userId

Reply:
My bad, thanks! Fixed